test/test_browser.py

```diff
@@ -990,8 +990,8 @@ def post():
       html = html.replace('</body>', '''
 <script src='fake_events.js'></script>
 <script>
-simulateKeyDown(1250);simulateKeyDown(38);simulateKeyUp(38);simulateKeyUp(1250); // alt, up
-simulateKeyDown(1248);simulateKeyDown(1249);simulateKeyDown(40);simulateKeyUp(40);simulateKeyUp(1249);simulateKeyUp(1248); // ctrl, shift, down
+simulateKeyDown(18);simulateKeyDown(38);simulateKeyUp(38);simulateKeyUp(18); // alt, up
+simulateKeyDown(17);simulateKeyDown(16);simulateKeyDown(40);simulateKeyUp(40);simulateKeyUp(16);simulateKeyUp(17); // ctrl, shift, down
```

Comment:
I'm not sure how to understand this change?

Reply:
I tried to describe this in the PR description.

The old code is passing raw SDL key values here (1248, 1249, 1250 which correspond to SDLK_LCTRL, SDLK_LSHIFT, SDLK_LALT).

This means that resulting DOM keyboard event contains SDL keybaord codes, which doesn't make any sense.  However because the SDL code was just copying any keyboard codes it didn't know about directly from the DOM event to the SDL event this was magically working.. but it should not have been.

The new key codes are the correct key codes from the corresponding DOM keys.

Reply:
Now I see, thanks!